js/shared.js

```diff
@@ -273,7 +276,7 @@ function playPattern(soundLength,buffer,pitch=PITCH_BIAS,
 	var step = freq / audio.sampleRate;
 	var pos = sampleState.pos;
 
-	var quality = 8;
+	var quality = Math.ceil( 192000 / audio.sampleRate ) * 2;
```

Comment:
I was noticing stuttering when setting the environment's sample rate 192000hz

The changes here is to compensate the super sampling quality according the environment's sample rate, this also to make the lowpass' cutoff frequency stays same on every environment sample rate's configuration.